### PR TITLE
TWIA 15

### PR DESCRIPTION
**Pull request by AnneKitsune**
Just the initial content. Feel free to push over my branch to add content or fix issues. I don't want to be spending too much time on this. ^^

### CODE REVIEW

File: src/content/blog/twia-15.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+* Recent changes
+* Current work
+* Interesting discussions
+* Call for participation
+
+## Recent changes
+
+* [Callback Queue] allowing better support for async tasks. [#1125](https://github.com/amethyst/amethyst/pull/1125)
+* With the Laminar 0.1.0 release, [it is now integrated](https://github.com/amethyst/amethyst/pull/1137) into Amethyst.
```
Comment (LucioFranco) [inline src/content/blog/twia-15.md]: `[0.1.0](https://crates.io/crates/laminar)`

File: src/content/blog/twia-15.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+* A big goal of Amethyst is to be user-friendly. As such, we opened [an issue](https://github.com/amethyst/amethyst/issues/1152) to gather opinions on this subject.
+
+## Call for participation
+
+We love contributions from the community, so if you see something you'd like
+to work on, don't hesitate! We are going to help you and point you into the
+right direction if something doesn't work out at first.
+
+We have tasks with different difficulty. Make sure to choose easier ones at
+first, because hard issues can become quite frustrating, especially if you
```
Comment (LucioFranco) [inline src/content/blog/twia-15.md]: I feel this last line comes off a bit strange. Just state that we have beginner ones.

File: src/content/blog/twia-15.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
++++
+title = "This week in Amethyst 15"
+description = "What happened this week: asset loading, skyboxes, fonts, working groups"
```
Comment (torkleyy) [inline src/content/blog/twia-15.md]: TODO

File: src/content/blog/twia-15.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+Content of this issue:
+
+* Recent changes
+* Current work
+* Interesting discussions
+* Call for participation
+
+## Recent changes
+
+* [Callback Queue] allowing better support for async tasks. [#1125](https://github.com/amethyst/amethyst/pull/1125)
```
Comment (torkleyy) [inline src/content/blog/twia-15.md]: Missing link I think.
The usual link style as per the docs is `[something][abc]`, so the link would go below. I don't mind this style, just saying.

**Review by torkleyy — COMMENTED** [review]
Thank you for doing this :heart:

**Comment by AnneKitsune** [comment]
moxi: would be nice to have an explicit call for suggestions on editor UI

File: src/content/blog/twia-15.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+* With the Laminar 0.1.0 release, [it is now integrated](https://github.com/amethyst/amethyst/pull/1137) into Amethyst.
+* [The look-at method was fixed](https://github.com/amethyst/amethyst/pull/1142).
+* [The new logo by Ellie](https://github.com/amethyst/amethyst/pull/1143) has been integrated into the examples.
+* [Rendering 2D images](https://github.com/amethyst/amethyst/pull/1153) is now much easier thanks to the changes made by @happenslol, based on the suggestions from [this RFC](https://github.com/amethyst/rfcs/pull/5).
+
+## What's being worked on currently?
+
+* [Pausable Systems][wo1] by @udoprog
+* [Better support for dynamic images][wo2] by @Stoeoef
+* A working group on the ui and rendering refactor has been formed. You can follow the progress in the `#ui` channel on discord.
```
Comment (torkleyy) [inline src/content/blog/twia-15.md]: Which working group?

File: src/content/blog/twia-15.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+* [Callback Queue] allowing better support for async tasks. [#1125](https://github.com/amethyst/amethyst/pull/1125)
+* With the Laminar 0.1.0 release, [it is now integrated](https://github.com/amethyst/amethyst/pull/1137) into Amethyst.
+* [The look-at method was fixed](https://github.com/amethyst/amethyst/pull/1142).
+* [The new logo by Ellie](https://github.com/amethyst/amethyst/pull/1143) has been integrated into the examples.
+* [Rendering 2D images](https://github.com/amethyst/amethyst/pull/1153) is now much easier thanks to the changes made by @happenslol, based on the suggestions from [this RFC](https://github.com/amethyst/rfcs/pull/5).
+
+## What's being worked on currently?
+
+* [Pausable Systems][wo1] by @udoprog
```
Comment (torkleyy) [inline src/content/blog/twia-15.md]: These first three sections could all use some more... "motivational detail" if that's a term. Just write some details about it that arouse interest.

File: src/content/blog/twia-15.md (hunk 5 above)
Comment (AnneKitsune) [reply src/content/blog/twia-15.md]: ui team. Those helping aren't officially part of the team yet however.

Comment (torkleyy) [reply src/content/blog/twia-15.md]: Hmm. If we write anything like that it should be official (and I still don't even know about it :sweat_smile:)

File: src/content/blog/twia-15.md (hunk 3 above)
Comment (torkleyy) [reply src/content/blog/twia-15.md]: This is not resolved yet.

Comment (LucioFranco) [reply src/content/blog/twia-15.md]: done

**Review by torkleyy — APPROVED** [review]
Thanks!

**Pull request merged**